TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/tests/test_views.py

```diff
@@ -0,0 +1,135 @@
+import os
+from django.test import Client, SimpleTestCase
+from moto import mock_aws
+from rest_framework import status
+from unittest.mock import patch
+
+
+VALID_CREATE_PAYLOAD = {
+    'PipelineRoleArn': 'arn:aws:iam::123456789012:role/pipeline-access-role',
+    'PipelineName': 'test-pipeline',
+    'AwsRegion': 'us-east-1',
+    'Tags': ['test-tag=123'],
+    'IndexRegexSelections': ['index-test-*', 'stored-*'],
+    'LogGroupName': '/aws/vendedlogs/osi-logs',
+    'SourceDataProvider': {
+        'Uri': 'http://vpc-test-source-domain.com:9200',
+        'AuthType': 'BASIC_AUTH',
+        'SecretArn': 'arn:aws:secretsmanager:123456789012:secret:source-secret'
+    },
+    'TargetDataProvider': {
+        'Uri': 'https://vpc-test-target-domain.com:443',
+        'AuthType': 'SIGV4'
+    },
+    'VpcSubnetIds': ['subnet-123456789', 'subnet-789012345'],
+    'VpcSecurityGroupIds': ['sg-123456789', 'sg-789012345']
+}
+VALID_UPDATE_PAYLOAD = {
+    'PipelineName': 'test-pipeline'
+}
+VALID_ASSUME_ROLE_UPDATE_PAYLOAD = {
+    'PipelineName': 'test-pipeline',
+    'PipelineManagerAssumeRoleArn': 'arn:aws:iam::123456789012:role/testRole',
+}
+
+
+# Moto has not yet implemented mocking for OSI API actions, using @patch for these calls
```

Comment:
So moto is basically handling/mocking creating the boto3 client?  It looks like we could add the OSI actions directly to moto (like https://docs.getmoto.org/en/latest/docs/services/patching_other_services.html or via PR to their repo), but I think this approach is fine with me for now